Source/Core/DolphinWX/CMakeLists.txt

```diff
@@ -212,6 +212,10 @@ if(wxWidgets_FOUND)
 		install(DIRECTORY ${BUNDLE_PATH} DESTINATION /Applications
 			USE_SOURCE_PERMISSIONS
 			)
+	elseif(WIN32)
+		set_target_properties(${DOLPHIN_EXE} PROPERTIES
+			WIN32_EXECUTABLE ON
```

Comment:
Ah, I mean that this property can be defined on non-Windows builds, and it won't affect anything: https://cmake.org/cmake/help/v3.0/prop_tgt/WIN32_EXECUTABLE.html "the executable _when linked on Windows_ will be created with a WinMain() entry point"